aea/cli/generate_wealth.py

```diff
@@ -84,30 +76,7 @@ def _try_generate_wealth(
                 address, testnet
             )
         )
-        try_generate_testnet_wealth(type_, address)
-        if sync:
-            _wait_funds_release(ctx.agent_config, wallet, type_)
+        try_generate_testnet_wealth(type_, address, sync)
 
     except ValueError as e:  # pragma: no cover
         raise click.ClickException(str(e))
-
-
-def _wait_funds_release(agent_config: AgentConfig, wallet: Wallet, type_: str) -> None:
```

Comment:
This does not work with the cosmos/fetchai approach as the faucet api does the syncing by default; it should also be the responsibility of the api to implement this, not the cli, so removed.